### ISSUE TITLE
Request apr and apr-util be included for Davinci Resolve

### ISSUE THREAD

**Issue by Zindswini** [ublue-os/main#91]
### Describe the package

[Davinci Resolve](https://www.blackmagicdesign.com/products/davinciresolve), a popular video editing/color suite requires two layered packages, `apr` and `apr-util`, but otherwise works out of the box on Silverblue using the official installer. It conveniently installs to /opt, which is tracked by ostree. (Lucky!)

These are small packages, and I believe that including them would streamline the linux experience for someone trying to follow the steps on the Resolve website.

If these are out of scope, I would instead like to add an entry to the website.

```
Dependencies resolved.
================================================================================================================================================
 Package                                 Architecture                  Version                              Repository                     Size
================================================================================================================================================
Installing:
 apr                                     x86_64                        1.7.0-16.fc36                        fedora                        124 k
 apr-util                                x86_64                        1.6.1-20.fc36                        fedora                         95 k
Installing weak dependencies:
 apr-util-bdb                            x86_64                        1.6.1-20.fc36                        fedora                         12 k
 apr-util-openssl                        x86_64                        1.6.1-20.fc36                        fedora                         15 k

Transaction Summary
================================================================================================================================================
Install  4 Packages

Total download size: 246 k
Installed size: 541 k
```

Thanks!

### Image

All Images

**Comment by castrojo** [comment]
Thanks for investigating this, is there anything special we should add on the website? Is the installer an RPM or a script? It might be worth a small set of instructions on the website if it's nonobvious.

**Comment by Zindswini** [comment]
The download page gives a `.run` executable which when ran opens an installer that eventually extracts the files into `/opt/resolve` after accepting terms. The installation of the main app worked fine, nothing in the installer seemed wrong besides a small theming error. 

I did encounter a bit of an issue, the `.desktop` files weren't put in place. There are placeholder files in `/opt/resolve/share/` which have "RESOLVE_INSTALL_LOCATION" instead of "/opt/resolve". I had to manually replace those and then copy them into `~/.local/share/applications`. This step could probably be simplified to a one-liner and put on the website.

Blackmagic also makes some hardware which the logs mention have udev rules but I don't really know how udev rules work and I don't have any of the devices to test with. FWIW the logs didn't mention that they failed to install. I also can't test the AMD side of this so if someone wants to give that a shot that might be helpful.

Overall though once I figured out how to fix the desktop file everything ran perfectly including (nvidia) hardware encoding. I was very impressed, everything "just works" for my specific configuration.

An entry on the website would probably just be outlining the install process, how to fix the desktop files, and how to set the correct file associations. There are also some general resolve+linux related things such as the AAC audio issue [Nobara talks about](https://nobaraproject.org/docs/davinci-resolve/my-recording-has-no-audio-when-imported/) on their docs. I hope this is helpful, if I can find time in the next week or two I'll try and submit a website PR and try to get that info added.

Details for reference:
Version: Davinci Resolve *Studio* 18.1.4
Ublue Image: bazzite-desktop
GPU: Nvidia Rtx 3080

Attached Install Log:
[resolveinstall.log](https://github.com/ublue-os/main/files/11130619/resolveinstall.log)
